fcrepo-exts/migration-utils
# Remove VersionAbstractionFedoraObjectHandler and Fedora2-specific code

Remove VersionAbstractionFedoraObjectHandler and Fedora2-specific code
* * *

**JIRA Ticket**: (None)

# What does this Pull Request do?
Removes Fedora2-specific content, and collapses 3 object handler interface/classes down to 2.

# How should this be tested?
Automated tests should pass. Any migrations you run (except from a Fedora2 repo) should run the same as previously.

# Interested parties
@fcrepo/committers
